autogen/cluster_regional.tf

```diff
@@ -20,13 +20,13 @@
   Create regional cluster
  *****************************************/
 resource "google_container_cluster" "primary" {
-  provider    = "{% if private_cluster %}google-beta{%else %}google{% endif %}"
+  provider    = "google-beta"
```

Comment:
[This section](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/autogen/README.md#terraform-and-plugins) of README.md will have to change as well, since it no longer is as described with this PR.